### PR TITLE
[DOCFIX] revise root level docs

### PR DESCRIPTION
**Pull request by Xenorith**
@aaudiber @apc999

### CODE REVIEW

File: docs/en/Getting-Started.md [hunk 1]
```diff
@@ -380,7 +349,7 @@ Alluxio can be deployed in many different environments.
 
 ### Under Storage Systems
 
-There are many Under storage systems that can be accessed through Alluxio.
+Various Under storage systems can be accessed through Alluxio.
```
Comment (Xenorith) [inline docs/en/Getting-Started.md]: fail at capitalization

File: docs/en/Overview.md [hunk 2]
```diff
@@ -10,76 +10,74 @@ priority: 0
...
-semantics that have performance implications compared to traditional file systems. For example, when
-accessing data in cloud storage there is no node-level locality or cross-application caching. There
-are also different performance characteristics in common file system operations like directory
-listing (‘ls’) and ‘rename’, which often add significant overhead to analytics. Deploying Alluixo
-with cloud or object storage can close the semantics gap and achieve significant performance gains.
+semantics that have performance implications compared to traditional file systems. Common file
+system operations such as directory listing and renaming often incur significant performance
+overhead. When accessing data in cloud storage, applications have no node-level locality or
+cross-application caching. Deploying Alluxio with cloud or object storage mitigates these problems
+because...TODO: how does Alluxio address this???
```
Comment (Xenorith) [inline docs/en/Overview.md]: this section explains the differences between object store vs filesystems but does not explain why alluxio helps

File: docs/en/Architecture-DataFlow.md [hunk 3]
```diff
@@ -53,27 +50,29 @@ Alluxio command-line, or the FUSE layer.
...
 </p>
 
-Alluxio master service can be deployed as one leader master and several standby
-masters for fault tolerance. When the leader master goes down, a standby master
-is elected to become the new leader master.
+The Alluxio master service can be deployed as one leading master and several standby
+masters for fault tolerance. When the leading master goes down, a standby master
+is elected to become the new leading master.
 
 #### Leader Master
```
Comment (apc999) [inline docs/en/Architecture-DataFlow.md]: leading master

File: docs/en/Architecture-DataFlow.md [hunk 4]
```diff
@@ -12,35 +12,32 @@ priority: 2
...
-  applications and frameworks running on top of Alluxio. Also, when mounting
-  multiple under storage systems simultaneously, Alluxio can serve as a unifying
-  layer for any number of varied data sources.
-
-At a high level, Alluxio can be divided into three components: the master,
-workers, and clients. Alluxio consists a single leader master, multiple standby masters,
-and multiple workers. The masters and workers together make up the Alluxio servers, which are
-the components a system admin would maintain and manage. The clients are used to
-talk to Alluxio servers by the applications, such as Spark or MapReduce jobs,
+Alluxio serves as a new data access layer in the ecosystem (TODO: what ecosystem???),
```
[the remark this answers is not in the document]
Comment (aaudiber) [reply docs/en/Architecture-DataFlow.md]: data analytics ecosystem? @apc999 wdyt?

File: docs/en/Architecture-DataFlow.md [hunk 5]
```diff
@@ -53,27 +50,29 @@ Alluxio command-line, or the FUSE layer.
...
 
-There is only one leader master in an Alluxio cluster. The leader master is
-responsible for managing the global metadata of the system. This includes file
-system metadata (e.g. the namespace tree), block metadata (e.g block locations),
-and worker capacity metadata (free and used space). Alluxio clients interact
-with the leader master to read or modify this metadata. In addition, all workers
-periodically send heartbeat information to the leader master to maintain their
+Only one master process can be the leader master in an Alluxio cluster.
+The leader master is responsible for managing the global metadata of the system.
+This includes file system metadata (e.g. the namespace tree), block metadata
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/Architecture-DataFlow.md]: maybe "filesystem inode tree" (linking to something like https://en.wikipedia.org/wiki/Inode_pointer_structure)

File: docs/en/Architecture-DataFlow.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/Architecture-DataFlow.md]: bigdata and machine learning ecosystem?

File: docs/en/Architecture-DataFlow.md [hunk 6]
```diff
@@ -53,27 +50,29 @@ Alluxio command-line, or the FUSE layer.
...
-with the leader master to read or modify this metadata. In addition, all workers
-periodically send heartbeat information to the leader master to maintain their
-participation in the cluster. The leader master does not initiate communication
-with other components; it only responds to requests via RPC
-services. Additionally, the leader master writes journals to a distributed
-persistent storage to allow for recovery of master state information.
+Only one master process can be the leading master in an Alluxio cluster.
+The leading master is responsible for managing the global metadata of the system.
+This includes file system metadata (e.g. the namespace tree), block metadata
+(e.g block locations), and worker capacity metadata (free and used space).
```
Comment (apc999) [inline docs/en/Architecture-DataFlow.md]: explain that, blocks are the basic unit of a file in Alluxio to store?

File: docs/en/Getting-Started.md [hunk 7]
```diff
@@ -252,14 +244,14 @@ dr-x------ staff  staff         4 PERSISTED 01-09-2018 16:34:55:362  DIR /mnt/s3
 -r-x------ staff  staff 157046046 PERSISTED 01-09-2018 16:35:01:002   0% /mnt/s3/sample_tweets_150m.csv
 ```
 
-This shows all the files under the root of the Alluxio file system from all of the mounted storage
+This shows all the files across all of the mounted storage
 systems. The `/LICENSE` file is from the local file system whereas the files under `/mnt/s3/` are
-from S3.
```
Comment (Xenorith) [inline docs/en/Getting-Started.md]: uh this should be from still? from local file system vs from S3?? @aaudiber

File: docs/en/Overview.md [hunk 8]
```diff
@@ -11,8 +11,8 @@ priority: 0
 ## What is Alluxio
 
 Alluxio is the world’s first memory-speed virtual distributed storage system.
-It unifies data access and bridges computation frameworks and their underlying storage systems.
-Applications can access data stored in numerous storage system through a single unified source.
+It bridges the gap between computation frameworks and storage systems, enabling applications
```
Comment (Xenorith) [inline docs/en/Overview.md]: removed "unifies data access" here? but it sounds so important :O @aaudiber

**Comment by Xenorith** [comment]
appreciate the updates, tweaked a bit more and left 2 questions

File: docs/en/Getting-Started.md (hunk 7 above)
Comment (aaudiber) [reply docs/en/Getting-Started.md]: I think both should be "in"

The file metadata is in both Alluxio and S3, but the important part here is that Alluxio is letting us transparently view S3 files in the same namespace as files in the local filesystem. Alluxio isn't the owner of the files (data is 0% loaded), so the files aren't "from" S3/Local filesystem. They are still in S3/Local filesystem, Alluxio is just giving a view of them.

File: docs/en/Overview.md (hunk 8 above)
Comment (aaudiber) [reply docs/en/Overview.md]: ugh

Comment (aaudiber) [reply docs/en/Overview.md]: I think it's unclear a less clear way of saying "enabling applications to connect to numerous storage systems through a common interface"

**Pull request merged**